config/grad.uiowa.edu/field.storage.node.field_person_scholar_srop_year.yml

```diff
@@ -0,0 +1,20 @@
+uuid: c7a605c0-cb93-4274-bb68-3f733f391648
+langcode: en
+status: true
+dependencies:
+  module:
+    - node
+    - taxonomy
+id: node.field_person_scholar_srop_year
+field_name: field_person_scholar_srop_year
+entity_type: node
+type: entity_reference
```

Comment:
Regarding SROP session year, I would prefer to not use taxonomy for this field. I'm open to either a list field with a PHP function to provide the list values (for keeping the year current) or potentially a date field that is just the year. The first one is probably less complicated.

This will allow building a view to display the results and not require the extra overhead (maintenance) of the vocab/taxonomy.

Reply:
I changed the SROP session year to the list field option. Thank you so much!